cumulusci/cli/cci.py

```diff
@@ -1285,6 +1286,7 @@ def org_shell(runtime, org_name, script=None, python=None):
         "help": CCIHelp(),
         "query": sf_helpers.query,
         "describe": sf_helpers.describe,
+        "record_count": sf_helpers.record_count,
     }
```

Comment:
Not a blocker for this PR, but I'm starting to think we should attach all this stuff to SimpleSalesforceUIHelpers, rename it to something like ShellKeywords, and use its `__dict__` as the globals.